webapp/src/main/webapp/news-list-view/components/settings/NewsSettingsDrawer.vue

```diff
@@ -138,11 +138,7 @@ export default {
       return this.viewTemplates.filter(e=> !e.name.includes('EmptyTemplate'));
     },
     checkAlphanumeric() {
-      if (this.newsHeader && !this.newsHeader.trim().match(/^[\w\-\s]+$/) && this.newsHeader.length > 0) {
-        return this.$t('news.list.settings.name.errorMessage');
-      } else {
-        return '';
-      }
+      return this.targetName && !this.targetName.trim().match(/^[\w\-\s]+$/) && this.targetName.length > 0 ? this.$t('news.list.settings.name.errorMessage') : '';
```

Comment:
this modification is a part of develop one https://github.com/exoplatform/news/commit/2c66a5445554b672c7426a42eda0ff810d026242#diff-7853b826a9d2c891b72fcf7bb148c5fde29d6335e95dc4c3a64b1cae755ccab2R104, so to be reverted